localstack/utils/bootstrap.py

```diff
@@ -360,6 +361,56 @@ def start_infra_locally():
     return infra.start_infra()
 
 
+def validate_localstack_config(name):
+    dirname = os.getcwd()
+    compose_file_name = '%s/%s' % (dirname, name)
+
+    # validating docker-compose file
+    cmd = "docker-compose -f '%s' config" % (compose_file_name)
+    try:
+        run(cmd)
+    except Exception:
+        return 'docker-compose file is not valid'
+
+    # validating docker-compose variable
+    with open(compose_file_name) as file:
+        compose_content = yaml.full_load(file)
+        localstack_service = [service for service in compose_content['services']
```

Comment:
nit: indentation can be reduced one level (moving the code to the left) for lines 378-384 (the `with` block is only required for the YAML loading)